### DISCUSSION TITLE
Support json5 or comment-json for Self Hosted Configuration

### DISCUSSION THREAD

**Discussion by ypresto** [renovatebot/renovate#10396] [Suggest an Idea]
It'd be better if we can write comment to Self Hosted Configuration.

I tried to rename it to `renovate.json5`, but didn't work :joy:.
So I want to write it explicitly in the document if it continues.

**Comment by rarkins** [comment]
@ypresto please stop creating Issues - especially ones without our templates - and instead create discussion issues if you need. Future issues will be deleted without notice.

**Comment by rarkins** [comment]
Which "self-hosted configuration" do you mean? For self-hosted the default filename is `config.js`, which allows comments.

**Comment by ikesyo** [comment]
This should be addressed in https://github.com/renovatebot/renovate/pull/12644.
